webpack.config.js

```diff
@@ -38,9 +38,6 @@ if (process.env.SENTRY_EXTRACT_TRANSLATIONS === '1') {
 var entry = {
   // js
   'app': 'app',
-  'translations': [
-    'app/translations'
-  ],
```

Comment:
This was not being used; everything is referenced from 'app' / pulled into dist/app.js